frida_tools/tracer.py

```diff
@@ -345,15 +345,39 @@ def _create_resolver_script(self):
     return workingSet;
 }
 
+function parseExportsFunctionPattern (pattern) {
```

Comment:
Let's move this function below where it's used, [newspaper style](https://kentcdodds.com/blog/newspaper-code-structure).